assets/js/hooks/useQueryString.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * `useQueryString` hook tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { renderHook, actHook } from '../../../tests/js/test-utils';
+import useQueryString from './useQueryString';
+
+describe( 'useQueryString', () => {
+	let oldLocation;
+	let oldHistory;
+	const historyPushMock = jest.fn();
```

Comment:
Let's name this to match the method more closely. Otherwise it might be confused as a mock for `history.pushState`.

```suggestion
	const historyReplaceStateMock = jest.fn();
```